TWLight/view_mixins.py

```diff
@@ -118,6 +119,46 @@ def dispatch(self, request, *args, **kwargs):
 
 
 
+class PartnerCoordinatorOnly(object):
```

Comment:
Seems like we could have this and the coordinator or self class share more code. Perhaps a shared function for partner coordinators or maybe a class with a few subclasses?